content/blog/2016/2016-05-10-jenkins-20-vjam.adoc

```diff
@@ -0,0 +1,355 @@
+---
+layout: post
+author: rtyler
+title: "Jenkins 2.0 Virtual JAM Wrap-up"
+tags:
+- jenkins2
+- jam
+- meetup
+---
+
+
+Last week we hosted our first ever
+link:http://www.meetup.com/Jenkins-online-meetup/[Virtual JAM] with the debut
+topic of: link:/2.0/[Jenkins 2.0]. link:https://github.com/alyssat[Alyssa], our
+Events officer, and I pulled together a
+link:http://www.meetup.com/Jenkins-online-meetup/events/230154841/[series of
+sessions] focusing on some of the most notable aspects of Jenkins 2 with:
+
+* A Jenkins 2.0 keynote from project founder
+ link:https://github.com/kohsuke[Kohsuke Kawaguchi]
+* An overview of "Pipeline as Code" from link:https://github.com/hrmpw[Patrick
+  Wolf]
+* A deep-dive into Pipeline and related plugins like Multibranch, etc from
+  link:https://github.com/jglick[Jesse Glick] and
+  link:https://github.com/kishorebhatia[Kishore Bhatia]
+* An overview of new user experience changes in 2.0 from
+  link:https://github.com/kzantow[Keith Zantow]
+* A quick lightning talk about documentation by yours truly
+* Wrapping up the sessions, was Kohsuke again, talking about the road beyond
+  Jenkins 2.0 and what big projects he sees on the horizon.
+
+++++
+<center>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/emV60CcDVV0" frameborder="0" allowfullscreen></iframe>
+</center>
+++++
+
+
+The event was really interesting for me, and I hope informative for those who
+participated in the live stream and Q&A session. I look forward to hosting more
+Virtual JAM events in the future, and I hope you will
+*link:http://www.meetup.com/Jenkins-online-meetup/[join us!]*
+
+=== Questions and Answers
+
+Below are a collection of questions and answers, that were posed during the
+Virtual JAM. Many of these were answered during the course of the sessions, but
+for posterity all are included below.
+
+==== Pipeline
+
+____
+What kind of DSL is used behind pipeline as code? Groovy or allow freely use
+different languages as a user prefer?
+____
+
+Pipeline uses a Groovy-based domain specific language.
+
+____
+How do you test your very own pipeline DSL?
+____
+
+Replay helps in testing/debugging while creating pipelines and at the branch
+level. There are some ideas which link:https://github.com/jglick[Jesse Glick]
+has proposed for testing `Jenkinsfile` and Pipeline libraries captured in
+link:https://issues.jenkins-ci.org/browse/JENKINS-33925[JENKINS-33925].
+
+____
+Isn't "Survive Jenkins restart" exclusive to Jenkins Enterprise?
+____
+
+No, this feature does not need CloudBees Jenkins Enterprise. All features shown
```

Comment:
Seems https://www.cloudbees.com/products/cloudbees-jenkins-platform is the best link